lib/src/layout/sidebar_item.dart

```diff
@@ -0,0 +1,502 @@
+import 'package:flutter/foundation.dart';
+
+import 'package:macos_ui/macos_ui.dart';
+import 'package:macos_ui/src/library.dart';
+import 'package:macos_ui/src/theme/macos_colors.dart';
+
+const Color _transparent = const Color(0x00000000);
```

Comment:
Let's add this to `MacosColors. Been meaning to do that.

Reply:
Not on this pr

Reply:
@bdlukaa was this done? Or are we doing it after?

Reply:
yeah, I did it